components/raftstore/src/store/peer.rs

```diff
@@ -1721,24 +1739,32 @@ where
                     self.raft_group.skip_bcast_commit(true);
                     self.last_urgent_proposal_idx = u64::MAX;
                 }
+                let current_term = self.term();
+                let cbs = committed_entries
```

Comment:
How about checking if `self.proposals` is empty first?

Reply:
It is checked at

https://github.com/tikv/tikv/blob/9b8785a42f6e03621c0d0ce66799761b3e00f9cc/components/raftstore/src/store/peer.rs#L1717